action.yml

```diff
@@ -3,7 +3,7 @@ description: 'Execute lint check on Bash scripts'
 author: 'ep-mobile'
 inputs:
   path:
-    description: 'Path to the bash script that needs the lint check.'
+    description: 'Path to the bash scripts that need the lint check.'
     required: true
```

Comment:
@Azbagheri, let's set `required: true` to `required: false` as we have support for linting all the files

Reply:
oh yeah, I totally missed that. Thanks, going to change it.